### PR TITLE
WinUI-2902 : Changed the content in the rating and busyindicator control.

### PR DESCRIPTION
**Pull request by Priyanka-Vijayasankar**
### Description:
Changed the content in the rating and busy indicator control.

### CODE REVIEW

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

File: winui/BusyIndicator/Overview.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: About WinUI BusyIndicator control| Syncfusion
+title: About WinUI BusyIndicator control | Syncfusion
 description: Learn here all about introduction of Syncfusion WinUI BusyIndicator Control (SfBusyIndicator) with key features and more.
```
Comment (OnyimbiNelson) [inline winui/BusyIndicator/Overview.md:4]: description: Learn all about the introduction of Syncfusion WinUI BusyIndicator Control (SfBusyIndicator) with essential features and more here.

File: winui/Rating/Overview.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: About WinUI Rating control| Syncfusion
+title: About WinUI Rating control | Syncfusion
 description: Learn here all about introduction of Syncfusion WinUI Rating Control (SfRating) with key features and more.
```
Comment (OnyimbiNelson) [inline winui/Rating/Overview.md:4]: description: Learn all about the introduction of Syncfusion WinUI Rating Control (SfRating) with essential features and more here.

File: winui/Rating/Precision.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ documentation: ug
 
 # Precision in WinUI Rating
 
-The rating value can be provided with three options such as full, half, and exact using the `Precision` property. The default value is `Full`.
+The Rating control provides option to set an accuracy level for ratings with flexible precision to handle full, half, or exact values using the `Precision` property. The default value is `Full`.
```
Comment (OnyimbiNelson) [inline winui/Rating/Precision.md:12]: The Rating control provides an option to set an accuracy level for ratings with flexible precision to handle full, half, or exact values using the `Precision` property. The default value is `Full`.

File: winui/BusyIndicator/Overview.md (hunk 1 above)
Comment (Priyanka-Vijayasankar) [reply winui/BusyIndicator/Overview.md:4]: Changed the overview content as mentioned above.

File: winui/Rating/Overview.md (hunk 2 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Overview.md:4]: Changed the description content as mentioned above.

File: winui/Rating/Precision.md (hunk 3 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Precision.md:12]: Changed the content as mentioned above.

**Pull request merged**